xbox_controller/launch/xbox_controller.launch

```diff
@@ -0,0 +1,19 @@
+<launch>
+	<!-- joy node -->
+		<node respawn="true" pkg="joy"
+			type="joy_node" name="turtle_joy">
+			<param name="dev" type="string" value="/dev/input/js0"/>
+			<param name="deadzone" value="0.2"/>
```

Comment:
A lil high

Reply:
put it at 0.1 or maybe lower? leaving at 0.1 for now then